scripts/release/update-peer-dependency-versions.js

```diff
@@ -3,7 +3,9 @@ const { modifyFile } = require('../lib/files-utils')
 const { command } = require('../lib/command')
 const { browserSdkVersion } = require('../lib/browser-sdk-version')
 
-const JSON_FILES = ['rum', 'rum-slim', 'logs'].map((packageName) => `./packages/${packageName}/package.json`)
+const JSON_FILES = ['rum', 'rum-slim', 'logs', 'rum-react'].map(
+  (packageName) => `./packages/${packageName}/package.json`
```

Comment:
**💭 thought:**  I wonder if we could avoid hardcoding package names here, like using `fs.readDirSync('packages')`‏